doc/faq.rst

```diff
@@ -254,17 +254,19 @@ specifying the pillar variable is the same one used for :py:func:`pillar.get
     <salt.states.file.managed>` state is only supported in Salt 2015.8.4 and
     newer.
 
+.. _faq-restart-salt-minion:
+
 What is the best way to restart a Salt Minion daemon using Salt after upgrade?
 ------------------------------------------------------------------------------
 
 Updating the ``salt-minion`` package requires a restart of the ``salt-minion``
 service. But restarting the service while in the middle of a state run
 interrupts the process of the Minion running states and sending results back to
 the Master. A common way to workaround that is to schedule restarting of the
-Minion service using :ref:`masterless mode <masterless-quickstart>` after all
-other states have been applied. This allows the minion to keep Minion to Master
-connection alive for the Minion to report the final results to the Master, while
-the service is restarting in the background.
+Minion service in background by issuing ``salt-call`` command calling
```

Comment:
I think `in background by issuing` --> `in the background by issuing a` is more readable. (Note the `the` and the `a`.)

I know the above line wasn't yours, but since we're here, can you also update `restarting of the` --> `restarting the`?

Reply:
Oh, those English articles! :smiley: